sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -21,65 +21,70 @@ Sets or updates one or more properties' values for a site collection.
- [-SharingAllowedDomainList <String>]
- [-SharingBlockedDomainList <String>]
- [-SharingDomainRestrictionMode <SharingDomainRestrictionModes>]
- [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>]
- [-StorageQuotaReset]
- [-DefaultSharingLinkType]
- [-DefaultLinkPermission]
- [-DefaultLinkToExistingAccess]
- [-ConditionalAccessPolicy <SPOConditionalAccessPolicyType>]
- [-AuthenticationContextName <String>]
- [-LimitedAccessFileType <SPOLimitedAccessFileType>]
- [-AllowEditing <Boolean>]
- [-AnonymousLinkExpirationInDays <Int32>]
- [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>]
- [-OverrideTenantExternalUserExpirationPolicy <Boolean>]
+ [-ExcludeBlockDownloadPolicySiteOwners <Boolean>]
+ [-ExcludedBlockDownloadGroupIds [Guid[]]]
  [-ExternalUserExpirationInDays <Int32>]
- [-SensitivityLabel <String>]
- [-RequestFilesLinkExpirationInDays <Int32>]
- [-RequestFilesLinkEnabled <Boolean>]
- [-RemoveLabel]
- [-BlockDownloadPolicy <Boolean>]
- [-OverrideBlockUserInfoVisibility <String>]
- [-OverrideSharingCapability <Boolean>]
- [-DefaultShareLinkScope <SharingScope>]
- [-DefaultShareLinkRole <SharingRole>]
  [-HidePeoplePreviewingFiles <Boolean>]
  [-HidePeopleWhoHaveListsOpen <Boolean>]
- [-AddRestrictedAccessControlGroups <Guid>]
- [-ClearRestrictedAccessControl <Boolean>]
- [-RestrictedAccessControl <Boolean>]
- [-RestrictedAccessControlGroups <Guid>]
- [-RemoveRestrictedAccessControlGroups <Guid>]
- [-ReadOnlyForUnmanagedDevices <Boolean>]
- [-ExcludedBlockDownloadGroupIds <Guid>]
- [-ExcludeBlockDownloadPolicySiteOwners <Boolean>]
- [-ReadOnlyForBlockDownloadPolicy <Boolean>]
- [-AuthenticationContextAccessType <SPOAuthenticationContextPolicyAccessType>]
  [-HubSiteId <Guid>]
+ [-Identity <SpoSitePipeBind>]
  [-InformationBarriersMode <String>]
+ [-LimitedAccessFileType <SPOLimitedAccessFileType>]
+ [-LocaleId <UInt32>]
+ [-LockState <String>]
+ [-LoopDefaultSharingLinkRole <SharingRole>]
+ [-LoopDefaultSharingLinkScope <SharingScope>]
+ [-MediaTranscription <MediaTranscriptionPolicyType>]
+ [-OverrideBlockUserInfoVisibility <String>]
+ [-OverrideSharingCapability <Boolean>]
+ [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>]
+ [-OverrideTenantExternalUserExpirationPolicy <Boolean>]
+ [-Owner <String>]
+ [-ReadOnlyForBlockDownloadPolicy <Boolean>]
+ [-ReadOnlyForUnmanagedDevices <Boolean>]
+ [-RemoveLabel <SwitchParameter>]
+ [-RemoveRestrictedAccessControlGroups [Guid[]]]
+ [-RequestFilesLinkEnabled <Boolean>]
+ [-RequestFilesLinkExpirationInDays <Int32>]
+ [-ResourceQuotaWarningLevel <Double>]
  [-RestrictContentOrgWideSearch <Boolean>]
  [-RestrictedAccessControl <Boolean>]
- [-RestrictedAccessControlGroups <Guid[]>]
- [-AddRestrictedAccessControlGroups <Guid[]>]
- [-RemoveRestrictedAccessControlGroups <Guid[]>]
- [-ClearRestrictedAccessControl <SwitchParameter>]
+ [-RestrictedAccessControlGroups [Guid[]]]
+ [-RestrictedToGeo <RestrictedToRegion>]
+ [-SandboxedCodeActivationCapability <SandboxedCodeActivationCapabilities>]
```

Comment:
We should not expose this property. This is internal only.

Reply:
After digging a little further, this seems an old feature started in 2016. Let's leave it as is then.